fairlearn/metrics_plots/plot_disparities_in_metric.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mtick
+from fairlearn.metrics import MetricFrame
+
+
+def plot_disparities_in_metric(metric, y_true, y_pred, sensitive_features, show_plot=True):
+    """Plot disparities in a metric between groups defined by sensitive features.
+
+    This method visualizes the metric values for all groups as identified
+    by sensitive features.
+
+    Parameters
+    ----------
+    metric : callable
+        A metric function that takes arguments `y_true` and `y_pred`
+    y_true : array-like
+        The list of true values
+    y_pred : array-like
+        The list of predicted values
+    sensitive_features : array-like
+        the sensitive attributes
```

Comment:
do we want to also support conditional features in plots?

Reply:
Yes, but not in this PR. I've opened #668 for this purpose. Should be a good task for a new contributor, for example. Otherwise I'll see whether I can get to it at some point soon.